### PR TITLE
align and lineHeight

### PR DESCRIPTION
**Pull request by tarah-s**
Hi, 

This is a neat component and works really well 👍  💯  good job.
I noticed an issue with text align and I added in line-height also because it's something I needed.
I was going to create ticket issue but thought since I'd done it already i'll just create a pull request.
Hope you don't mind.

Cheers, 
Tarah

### CODE REVIEW

**Comment by bryik** [comment]
Thanks for adding line height and fixing align! 

I'll look this over.

File: examples/line-height/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      </a-entity>
+
+      <a-entity position="0 2 0"
+                bmfont-text="text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo nunc vestibulum mollis convallis.;
+                            color: white; width: 500; height: 300; lineHeight: 55; ">
+
+      <a-entity position="0 2 0"
+                bmfont-text="text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo nunc vestibulum mollis convallis.;
+                            color: white; width: 500; height: 300;  ">
+      </a-entity>
```
Comment (bryik) [inline examples/line-height/index.html:22]: The "height: 300" properties aren't needed, right?

Comment (tarah-s) [inline examples/line-height/index.html:22]: Ah yeah Sorry I was testing out something else and didn't remove

**Comment by bryik** [comment]
No problem. 

Could you remove them and [update your pull request?](http://stackoverflow.com/questions/9790448/how-to-update-a-pull-request)

**Comment by bryik** [comment]
Actually this is quick enough to do myself. 

Thanks for your help!

**Pull request merged**